velox/docs/functions/spark/math.rst

```diff
@@ -67,6 +67,15 @@ Mathematical Functions
 
     Returns the natural logarithm of the “given value ``x`` plus one”.
     Return NULL if x is less than or equal to -1.
+
+.. spark:function:: log2(x) -> double
+
+    Returns the logarithm of ``x`` with base 2. Return null for zero input.
```

Comment:
Also can you update the doc to say return null for non-positive input?

Reply:
Thanks, Just Updated.